src/report/api/serializers.py

```diff
@@ -20,6 +22,28 @@ class ValidateUserAccessLabRoomSerializer(ValidateUserAccessOrgLabSerializer):
         queryset=ShelfObject.objects.using(settings.READONLY_DATABASE), allow_null=True,
         required=False)
 
+    def validate(self, data):
+        data = super().validate(data)
+        laboratory = data['laboratory']
+        organization = data['organization']
+        lab_room = data['lab_room'] if "lab_room" in data else None
+
+        if lab_room:
+            if lab_room.laboratory != laboratory:
+                logger.debug(
+                    f'ValidateUserAccessLabRoomSerializer --> labroom.laboratory != laboratory')
```

Comment:
In ValidateUserAccessLabRoomSerializer serializer class is necessary, check if lab_room element is an object or a list, if it a list avoid the laboratory validation.